src/javascripts/views/homeView.js

```diff
@@ -77,7 +75,19 @@ class HomeView {
         this.summaryTabEl.innerHTML = summaryTemplate;
     }
 
+    
     //----- EVENT HANDLER -----//
+    addDelegateShowInfo = (handlerTransactionInfo) => {
+        this.transactionListEl = this.categoryListEl.querySelector(".transaction-list");
+        this.transactionListEl.addEventListener("click", (event) => {
+            const el = event.target.closest(".transaction-detail");
```

Comment:
No need empty line between const.
Should define meaning variable.

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!